### ISSUE TITLE
snap/snapcraft.yaml is malformed

### ISSUE THREAD

**Issue by cjwatson** [Manisso/fsociety#66]
```
$ curl -s https://raw.githubusercontent.com/Manisso/fsociety/master/snap/snapcraft.yaml | \
  python3 -c 'import sys, yaml; yaml.safe_load(sys.stdin.read())'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python3/dist-packages/yaml/__init__.py", line 94, in safe_load
    return load(stream, SafeLoader)
  File "/usr/lib/python3/dist-packages/yaml/__init__.py", line 72, in load
    return loader.get_single_data()
  File "/usr/lib/python3/dist-packages/yaml/constructor.py", line 35, in get_single_data
    node = self.get_single_node()
  File "/usr/lib/python3/dist-packages/yaml/composer.py", line 39, in get_single_node
    if not self.check_event(StreamEndEvent):
  File "/usr/lib/python3/dist-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/usr/lib/python3/dist-packages/yaml/parser.py", line 174, in parse_document_start
    self.peek_token().start_mark)
yaml.parser.ParserError: expected '<document start>', but found '<scalar>'
  in "<unicode string>", line 7, column 1:
    A Penetration Testing Framework, ...
    ^
```

This causes build requests to fail on build.snapcraft.io.  It looks like you just need to fix up the weirdly-inverted indentation.

**Comment by CRO-THEHACKER** [comment]
Yeah, that ";" Is the problem... Why did Manisso do that 😂

On Nov 18, 2018 8:24 AM, "Colin Watson" <notifications@github.com> wrote:

$ curl -s https://raw.githubusercontent.com/Manisso/fsociety/master/snap/snapcraft.yaml
| python3 -c 'import sys, yaml; yaml.safe_load(sys.stdin.read())'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python3/dist-packages/yaml/__init__.py", line 94, in safe_load
    return load(stream, SafeLoader)
  File "/usr/lib/python3/dist-packages/yaml/__init__.py", line 72, in load
    return loader.get_single_data()
  File "/usr/lib/python3/dist-packages/yaml/constructor.py", line 35,
in get_single_data
    node = self.get_single_node()
  File "/usr/lib/python3/dist-packages/yaml/composer.py", line 39, in
get_single_node
    if not self.check_event(StreamEndEvent):
  File "/usr/lib/python3/dist-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/usr/lib/python3/dist-packages/yaml/parser.py", line 174, in
parse_document_start
    self.peek_token().start_mark)
yaml.parser.ParserError: expected '<document start>', but found '<scalar>'
  in "<unicode string>", line 7, column 1:
    A Penetration Testing Framework, ...
    ^

This causes build requests to fail on build.snapcraft.io. It looks like you
just need to fix up the weirdly-inverted indentation.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/Manisso/fsociety/issues/66>, or mute the thread

**Comment by cjwatson** [comment]
> Yeah, that ";" Is the problem... Why did Manisso do that

There is no ";" in the file.  The problem is that the indentation is all wrong.

**Comment by CRO-THEHACKER** [comment]
Can you nano it so I can see the file 😂

**Comment by cjwatson** [comment]
I filed #67.

**Comment by Manisso** [comment]
😂😂😂😂 Sorry  guys i am So Tired 
i added a YAML File and I forgot to edit it

**Comment by CRO-THEHACKER** [comment]
@Manisso OK, then add it :joy:

**Comment by cjwatson** [comment]
This is still an issue (as an operator of build.snapcraft.io, I got an error email about it yesterday).  If you aren't willing to actually fix it, should I just deactivate your repository on build.snapcraft.io for you?

**Comment by CRO-THEHACKER** [comment]
Hello, We can fix this!

Please give us time due too me working a full time job!

**Comment by Nikita11790** [comment]
hi?

**Comment by DamAg3K** [comment]
Where did u get my email address?

**Comment by Nikita11790** [comment]
Where did you get my email address?





> 
> ﻿Where did u get my email address?
>

**Comment by up2smoke** [comment]
Reminder +1

**Comment by CRO-THEHACKER** [comment]
Closing Error due to no fix...

**Comment by cjwatson** [comment]
@CRO-THEHACKER Huh?  There's a fix in #67.  I don't understand what's so hard about this; the YAML is invalid and fixing it isn't that complicated.
